{"uuid":"5c71f137-2a30-4ad9-9227-6fe53297e240","name":"Scenario: User enters Invalid Python code and Presented with Error Message","children":["b9a72af4-1119-42c2-8505-ed4fb0b5fdb8"],"befores":[],"afters":[{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687878293946,"stop":1687878293946},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687878294156,"stop":1687878294156},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687878294241,"stop":1687878294241},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687878294280,"stop":1687878294280},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687878294290,"stop":1687878294290}],"start":1687878293839,"stop":1687878294290}